index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>capstone-project</title>
+    <link rel="stylesheet" href="style.css"/>
+    <script src="https://kit.fontawesome.com/f25b74c76b.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header id="dsktop-header">
+        <nav id="nav1" title="first-navigation">
+          <ul>
+            <li><a class="fa-brands fa-facebook-f"></a></li>
+            <li><a class="fa-brands fa-twitter"></a></li>
+            <li><a href="#" class="nav1item">English</a></li>
+            <li><a href="#" class="nav1item">My Page</a></li>
+            <li><a href="#" class="nav1item">Logout</a></li>
+          </ul>
+        </nav>
+         
+        <nav id="nav2" title="second-navigation"> 
+          <a class="main-logo" href="#home"><img src="images/icecream.png" alt="logo"></a>
+          
+          <a class="nav2Item" href="./about.html">About</a>
+          <a class="nav2Item" href="#packages">Packages</a>
+          <a class="nav2Item" href="./about.html">Join</a>
+          <a class="nav2Item" href="#sponsors">Sponsor</a>
+          <a class="nav2Item" href="./about.html">Recipes</a>
+          <a class="mainItem" href="./about.html">Workshops</a> 
+        </nav>   
+    </header>
+
+    <header id="mobile-header">
+      <nav class="mobile-nav"> 
+          <ul onclick="menuPopup()" id="navList" title="navigation-list">
+            <li><a class="navItem" href="./about.html">About</a></li>
+            <li><a class="navItem" href="#packages">Packages</a></li>
+            <li><a class="navItem" href="./about.html">Join</a></li>
+            <li><a class="navItem" href="#sponsors">Sponsor</a></li>
+            <li><a class="navItem" href="./about.html">Recipes</a></li>
+            <li><a class="navItem navItem-main" href="./about.html">Workshops</a></li>
+          </ul> 
+      </nav>   
+      <nav>
+        <div onclick="menuPopup()" id="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div> 
+        <div onclick="menuPopup()" id="closeButton"> &times; </div>
```

Comment:
- [ ] Just as I suggested above, It will be of good practice to move this functionality in the `main.js` file :+1: :smile: